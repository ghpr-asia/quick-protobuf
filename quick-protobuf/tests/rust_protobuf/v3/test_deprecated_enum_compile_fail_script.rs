// DO NOT link to module tree
// This file is run remotely from test_deprecated_enum.rs and should fail to compile

#[path = "./test_deprecated_enum_without_add_deprecated_fields.rs"]
pub mod proto;
use crate::proto::Thing;

fn main() {
    let t = Thing::C;
}
