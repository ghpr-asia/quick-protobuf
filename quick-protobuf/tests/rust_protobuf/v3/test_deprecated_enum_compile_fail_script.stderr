error[E0599]: no variant or associated item named `C` found for enum `Thing` in the current scope
 --> tests/rust_protobuf/v3/test_deprecated_enum_compile_fail_script.rs:9:20
  |
9 |     let t = Thing::C;
  |                    ^
  |                    |
  |                    variant or associated item not found in `Thing`
  |                    help: there is a variant with a similar name: `A`
  |
 ::: tests/rust_protobuf/v3/./test_deprecated_enum_without_add_deprecated_fields.rs
  |
  | pub enum Thing {
  | -------------- variant or associated item `C` not found for this enum
